### ISSUE TITLE
Footer contrast with light variant

### ISSUE THREAD

**Issue by igorkulman** [nanxiaobei/hugo-paper#163]
When using the light variant instead of linen the google page speed score for accessibility decreases from 100 to 97 because if insufficient contrast between the background and the footer text.

<img width="944" alt="Screenshot 2023-03-19 at 10 46 40" src="https://user-images.githubusercontent.com/4746070/226166849-cbec7126-c07d-4c76-9c22-1d0c12f140aa.png">

It would be nice to adjust the footer color in light mode to get the perfect score.

**Comment by nanxiaobei** [comment]
thanks for your feedback, updated to `6.10.0` and fixed it.
